gems/pending/util/mount/miq_generic_mount_session.rb

```diff
@@ -14,8 +14,8 @@ class MiqGenericMountSession
   attr_accessor :settings, :mnt_point, :logger
 
   def initialize(log_settings)
+    raise "URI missing" if log_settings[:uri].nil?
```

Comment:
How about `raise "URI missing" unless log_settings.key?(:uri)`

What are the chances someone will provide an explicit nil :uri?